data_pipeline/twitter_ads_api/twitter_ads_api_config.py

```diff
@@ -7,21 +7,63 @@
 )
 
 
+class TwitterAdsApiParameterValuesConfig(NamedTuple):
+    from_bigquery: BigQuerySourceConfig = {}
+    start_time_value: str = ''
+
+    @staticmethod
+    def from_dict(parameter_values_config_dict: dict) -> 'TwitterAdsApiParameterValuesConfig':
+        return TwitterAdsApiParameterValuesConfig(
+            from_bigquery=BigQuerySourceConfig.from_dict(
+                parameter_values_config_dict.get('fromBigQuery', {})
+            ),
+            start_time_value=parameter_values_config_dict.get('startTimeValue', '')
+        )
+
+
+class TwitterAdsApiParameterNamesForConfig(NamedTuple):
+    bigquery_value: str = ''
```

Comment:
Similar to a previous comment, I would generally mark them as `Optional[str]` and default to `None`. I believe a blank string is usually not a good default value.

In this iteration you could even argue that they are all mandatory?